pmm-app/tests/verifyRemoteInstances_test.js

```diff
@@ -27,6 +27,50 @@ Data(instances.filter((instance) => instance.name !== 'mongodb')).Scenario(
   },
 );
 
+Scenario(
+  'PMM-T588 - Verify adding external exporter service via UI @not-pr-pipeline',
```

Comment:
Good, @IvanaBrecherova please create a task to also add username/password based external service, so we cover authentication case also, please put it for grooming tomorrow.

Reply:
Also please add another case to make sure we are getting metrics from this external exporter, use data-exploration dashboard for such validation

Reply:
@puneet0191 Created: https://jira.percona.com/browse/PMM-7770 :)